metadata_mapper/mappers/mapper.py

```diff
@@ -137,9 +140,12 @@ def select_oac_id(self):
         else:
             calisphere_id = id_values[0]
 
-        self.mapped_data["id"] = f"{self.collection_id}--{calisphere_id}"
-        self.mapped_data["isShownAt"] = calisphere_id
-        self.mapped_data["isShownBy"] = f"{calisphere_id}/thumbnail"
+        self.pre_mapped_data["id"] = f"{self.collection_id}--{calisphere_id}"
+        self.pre_mapped_data["calisphere-id"] = f"{calisphere_id}"
+        self.pre_mapped_data["isShownAt"] = calisphere_id
+        self.pre_mapped_data["isShownBy"] = f"{calisphere_id}/thumbnail"
+        self.legacy_couch_db_id = (f"{self.collection_id}--{calisphere_id}")
```

Comment:
Currently the validation outputs the legacy couchdb id as a way to look up how the record looks in our existing system. Calculating this legacy couchdb id in rikolti helps us match against the existing system. Unlike our solr ids and "calisphere-id", I do understand how the couchdb id is calculated each time. I'd prefer to have both in the system until we can get rid of both solr and couch.

Reply:
ok, makes sense